Cargo.toml

```diff
@@ -13,20 +13,24 @@ members = ["examples"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
-default = ["async-tungstenite", "cynic"]
+default = ["async-tungstenite", "ws_stream_wasm", "cynic"]
 
 [dependencies]
-async_executors = { git = "https://github.com/obmarg/async_executors.git", branch = "tokio-context" }
-futures = { version = "0.3"}
+async_executors = { version = "0.4.2" }
```

Comment:
Did you specifically need to upgrade `async_executors` to get this to work?  I might go and update my fork if so - unfortunately it's needed for the tokio executor.

Reply:
No, I just like always updating to the newest and for my personal needs I did not care much about tokio :P